invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposits/search_app_customizations.js

```diff
@@ -0,0 +1,169 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2020 CERN.
+// Copyright (C) 2020 Northwestern University.
+//
+// Invenio App RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React from "react";
+import { overrideStore } from "react-overridable";
+import {
+  Icon,
+  Card,
+  Checkbox,
+  Grid,
+  Label,
+  Input,
+  Item,
+} from "semantic-ui-react";
+import _ from "lodash";
+import _truncate from "lodash/truncate";
+
+const RDMRecordResultsListItem = ({ result, index }) => {
+  const startDate = _.get(result, "metadata.dates[0].start", "No metadata");
+  const status = _.get(
+    result,
+    "metadata.resource_type.type",
+    "No resource type"
+  );
+  const access = _.get(
+    result,
+    "metadata._default_preview",
+    "No default preview"
+  );
+  const creatorName = _.get(result, "metadata.creators[0].name", "No creator");
+  const updatedDate = _.get(result, "metadata.embargo_date", "No updated date"); // FIXME: probably wrong
+  const title = _.get(result, "metadata.titles[0].title", "No title");
+  const author = _.get(result, "metadata._internal_notes[0].user", "anonymous");
+
+  return (
+    <Item key={index} className="deposits-list-item">
+      <Item.Content>
+        <Grid>
+          <Grid.Row columns={2}>
+            <Grid.Column width={1} className="checkbox-column">
+              <Checkbox />
+              {/* FIXME: prototype */}
+              {Math.floor(Math.random() * Math.floor(2)) ? (
+                <Icon name="check" color="green" />
+              ) : (
+                <Icon name="upload" color="red" />
+              )}
```

Comment:
Displaying randomly one icon or the other. Fix it when implementing it properly.